boards/fanke/fk750m1_vbt6/board.yml

```diff
@@ -0,0 +1,11 @@
+board:
+  name: fk750m1_vbt6
+  vendor: fanke
+  socs:
+    - name: stm32h750xx
+  revision:
+    format: major.minor.patch
+    default: 1.1.0
+    revisions:
+      - name: 1.0.0
+      - name: 1.1.0
```

Comment:
```suggestion
```

Reply:
So do we need to remove this code?

Reply:
you're not adding any version specific files to this PR, so yes. You can write in the documentation that it's for board version 1.1.0 or add the version files for 1.0.0 by checking the schematic differences but it doesn't make sense to have versions in board.yml that do nothing different and both build the exact same configuration

Reply:
I got it. Thanks a lot.